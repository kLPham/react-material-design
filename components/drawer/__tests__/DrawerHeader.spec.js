import React from 'react';
import { shallow } from 'enzyme';
import DrawerHeader from '../drawerHeader';

describe('<DrawerHeader />', () => {
    it('should render', () => {
        const drawerHeader = shallow(
            <DrawerHeader>Text</DrawerHeader>,
    );
        expect(drawerHeader).toMatchSnapshot();
    });
    it('should render primary', () => {
        const primary = true;
        const drawerHeader = shallow(
            <DrawerHeader primary={primary}>Text</DrawerHeader>,
    );
        expect(drawerHeader).toMatchSnapshot();
    });
});
