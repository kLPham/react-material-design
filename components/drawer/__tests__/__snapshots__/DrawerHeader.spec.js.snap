// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<DrawerHeader /> should render 1`] = `
<header
  className="mdc-temporary-drawer__header"
>
  <div
    className="mdc-temporary-drawer__header-content"
  >
    Text
  </div>
</header>
`;

exports[`<DrawerHeader /> should render primary 1`] = `
<header
  className="mdc-temporary-drawer__header"
>
  <div
    className="mdc-temporary-drawer__header-content mdc-theme--primary-bg mdc-theme--text-primary-on-primary"
  >
    Text
  </div>
</header>
`;
